lib/mba/mba.go

```diff
@@ -0,0 +1,38 @@
+package mba
+
+import (
+	"io/ioutil"
+	"strconv"
+	"strings"
+
+	"github.com/intel/rmd/lib/proc"
+)
+
+var mbaStep, mbaMin = -1, -1
```

Comment:
Hi Shihta,
Everything looks good to me. One minor change - please move var mbaStep, mbaMin = -1, -1 as local variables in GetMbaInfo

Reply:
Also can you raise an issue for the bugs you pointed out - A.B and C